experimental/uefi/loader/src/server.rs

```diff
@@ -28,6 +28,16 @@ use proto::{
     UnaryRequest, UnaryResponse,
 };
 
+fn serialize_request(request: UnaryRequest) -> Vec<u8> {
+    // The payload is the request's body prepended with the 8 byte session_id.
+    // This takes adavantage of the session_id's fixed size to avoid needing
+    // to use a key/value binary serialization protocol.
```

Comment:
As suggested by @tiziano88 in https://github.com/project-oak/oak/pull/2759#discussion_r856435136 :)